application/core/LSHttpRequest.php

```diff
@@ -286,4 +285,21 @@ public function isRestRequest()
         ) === 1;
         return $restPath || $restRoute;
     }
+
+    /**
+     * @inheritdoc
+     * Check host with config['allowedHost'] if it set
+     */
+    public function getHostInfo($schema = '')
+    {
+        $hostInfo = parent::getHostInfo($schema);
+        if (!empty(App()->getConfig('allowedHosts')) && is_array(App()->getConfig('allowedHosts'))) {
+            $allowedHosts = App()->getConfig('allowedHosts');
```

Comment:
Small Comment:

Shouldn't we pick up the variale before the if ?
That would aboid calling the same method 3 times.

Reply:
:+1: